addons/core/translations/resources/en-us.yaml

```diff
@@ -374,27 +374,30 @@ target:
     diagram:
       client: Client
       any-worker: Any worker
-      front-line-worker: Front line worker
+      frontline-worker: Frontline worker
       hcp-worker: HCP worker
       egress-worker: Egress worker
       ingress-worker: Ingress worker
       host: Host
       network: Private network
     filter-explainer:
-      any-worker: In this configuration, the client will attemt to connect through any client-accessible worker.
-      egress-worker: In this configuration, the client will attemt to connect through any filtered egress worker.
-      ingress-worker: In this configuration, the client will attempt to conect through any filtered ingress worker.
+      any-worker: In this configuration, the client will attempt to connect through any client-accessible worker.
+      egress-worker: In this configuration, the client will attempt to connect through any filtered egress worker.
+      ingress-worker: In this configuration, the client will attempt to connect through any filtered ingress worker.
       hcp-worker: In this configuration, the client will attempt to connect through any HCP-managed worker.
-      ent-dual-egress-on: In this configuration, the client will attempt to  connect through any client-accessible "frontline" worker and reach the host through any filtered egress worker.
-      hcp-dual-egress-on: In this configuration, the client will attempt to connect through any HC-managed worker and reach the host through any filtered egress worker.
-      dual-egress-on-ingress-on: In this configuration, the client will attempt to connect through any filtered ingress worker and reach the house through any filtered egress worker.
+      dual-egress-on: In this configuration, the client will attempt to  connect through any client-accessible "frontline" worker and reach the host through any filtered egress worker.
+      dual-egress-on-ingress-on: In this configuration, the client will attempt to connect through any filtered ingress worker and reach the host through any filtered egress worker.
+      hcp-dual-egress-on: In this configuration, the client will attempt to connect through any HCP-managed worker and reach the host through any filtered egress worker.
+    filter:
+      title: Filter
+      description: Boolean expression.
+      update_filter: Update Filter
```

Comment:
This seems out of place?  What is this about?  Non-field keys are dasherized.  And this seems like an action that would be organized accordingly.  How do we organize resource-specific actions elsewhere?

Reply:
Good point, I could move this to actions.yaml where we have other actions

Reply:
Moved it to an actions catagory similar to other resources.